.travis.yml

```diff
@@ -5,6 +5,7 @@ cache: bundler
 rvm:
 - jruby-1.7.25
 script:
-- bundle exec rspec spec
+- bundle exec rspec spec --order rand
```

Comment:
I believe this does not run the `--integration` tests by default.  We should change this to:

`bundle exec rspec && bundle exec rspec --tag integration`